common/armies/hfe_armies.txt

```diff
@@ -38,3 +38,306 @@
 # 		}
 # 	}
 # }
+
+hfe_army_high_mechanicus_bodyguards = {
+	defensive = yes
+	is_pop_spawned = no
+	pop_limited = no
+	health = 1.1
+	damage = 2
+	morale = 5
+	has_morale = yes
+	morale_damage = 1.5
+	collateral_damage = 0.9
+	war_exhaustion = 2
+	icon_frame = 5
+	has_species = yes
+
+	resources = {
+		category = armies
+		cost = {
+#			trigger = {
+#				owner = {
+#					NOT = {
+#						is_primitive = yes
+#						is_country_type = primitive
+#					}
+#				}
+#			}
+			energy = 500
+			food = 30
+			alloys = 300
+		}
+		cost = {
+			alloys = 100
+		}
+		produces = {
+			unity = 1
+		}
+		upkeep = {
+			food = 2
+			alloys = 2
+			energy = 2
+			minerals = 2
+		}
+	}
+
+	potential = {
+		owner = {
+			has_country_flag = hfe_varelse_primitive_0 
+
+			OR = {
+				has_civic = hfe_civic_high_mechanicus
+				has_country_flag = high_mechanicus_retained
+				is_spiritualist = yes
+			}
+			NOT = {
+				has_civic = hfe_civic_overthrown_mechanicus
+				has_country_flag = hfe_varelse_fedora_governance
+				is_materialist = yes
+			}
+		}
+		planet = {
+			OR = {
+				has_planet_flag = colony_beta
+				is_capital = yes
+			}
+		}
+	}
+	allow = {
+		planet = {
+			NOT = { # Only one can exist on a planet
+				any_planet_army = {
+					army_type = hfe_army_high_mechanicus_bodyguards
+				}
+			}
+		}
+		OR = {
+			FROM = {
+				OR = {
+					has_trait = trait_mechanical
+					has_trait = trait_machine_unit
+					is_robotic = yes
+				}
+			}
+			owner = { has_country_flag = hfe_varelse_primitive_0 }
+		}
+		OR = {
+			FROM = {
+				OR = {
+					has_citizenship_rights = yes
+					has_citizenship_type = { type = citizenship_full }
```

Comment:
This lines do same thing
has_citizenship_rights = yes
has_citizenship_type = { type = citizenship_full }

Reply:
So?

Reply:
Use only first (i gues it check other sitezenship types like full machine or refer to game rules).

Reply:
This is for expanded compatibility. What if Paradox or a Mod makes changes to either of these? With this code, we are prepared for anything!

Reply:
Single has_citizenship_rights = yes should have maximum compatibility with future changes, also it covers more than one citizenship type.

Reply:
This also covers for Mods and sets a guaranteed default. That is why I used both.